pysteps/blending/utils.py

```diff
@@ -557,3 +565,86 @@ def check_norain(precip_arr, precip_thr=None, norain_thr=0.0):
         f"Rain fraction is: {str(rain_pixels.size / precip_arr.size)}, while minimum fraction is {str(norain_thr)}"
     )
     return norain
+
+
+def compute_smooth_dilated_mask(
+    original_mask,
+    max_padding_size_in_px=0,
+    gaussian_kernel_size=9,
+    inverted=False,
+    non_linear_growth_kernel_sizes=False,
+):
+    """
+    Compute a smooth dilated mask using Gaussian blur and dilation with varying kernel sizes.
+
+    Parameters
+    ----------
+    original_mask : array_like
+        Two-dimensional boolean array containing the input mask.
+    max_padding_size_in_px : int
+        The maximum size of the padding in pixels. Default is 100.
+    gaussian_kernel_size : int, optional
+        Size of the Gaussian kernel to use for blurring. Default is 9.
+    inverted : bool, optional
+        If True, invert the original mask before processing. Default is False.
+    non_linear_growth_kernel_sizes : bool, optional
+        If True, use non-linear growth for kernel sizes. Default is False.
+
+    Returns
+    -------
+    initial_mask : array_like
+        The initial binary mask after Gaussian blur and thresholding.
+    new_mask : array_like
```

Comment:
It returns final_mask instead of the two variables mentioned above. Are they the same or different?